server/src/index.js

```diff
@@ -83,6 +83,18 @@ io.on('connection', (socket) => {
         socket.emit('gif_response', { response: false });
       });
   });
+
+  socket.on('save_gif', (blob) => {
```

Comment:
Saves the blob coming from the end of the p5 sketch, and then sends back "ok" or "not ok" to the sketch, which we can use to switch back to the camera.